### PR TITLE
Add placeholder text for the REPL

### PR DESCRIPTION
**Pull request by ryanford**
I added a placeholder telling users where to enter text. Re: non-obvious REPL issue #2

### CODE REVIEW

File: views/index.etlua [hunk 1]
```diff
@@ -15,7 +15,18 @@
         <img src="/static/images/github-icon.svg" alt="Github" width="32" class="logo">
     </a>
 </div>
-
+<script>
+  document.getElementById('fengari-input').addEventListener('focus', function() {
```
Comment (giann) [inline views/index.etlua:19]: I don't think a placeholder needs to be manually managed

Comment (ryanford) [reply views/index.etlua:19]: The default behavior is that it remains until input and snaps back if the input goes back to "". I added the js to draw more attention to the fact that it was an interactive/input element and just behaves a little nicer.

**Comment by daurnimator** [comment]
Okay to close? I think this was fixed in redesign.

**Comment by giann** [comment]
Yes, closing

**Pull request closed without merging**